src/gerbil/runtime/gx-gambc0.scm

```diff
@@ -1395,9 +1395,9 @@
            (lp rest (cons hd r))))
         (else lst)))))
 
-(define-remove remove equal?)
-(define-remove remv eqv?)
-(define-remove remq eq?)
+(define-remove1 remove1 equal?)
+(define-remove1 remv eqv?)
+(define-remove1 remq eq?)
```

Comment:
shouldn't the other variants also get a `1` in their name, then?

Reply:
we discussed this over email and I don't think they should -- they don't conflict with anything from srfi.